pom.xml

```diff
@@ -70,7 +69,6 @@
         <groupId>ch.qos.logback</groupId>
         <artifactId>logback-classic</artifactId>
         <version>${logback.version}</version>
-        <scope>test</scope>
```

Comment:
I'm not in favor of removing this -- this dependency should be used by tests, and only by tests, in **every** project in this repository.

Reply:
So it closes the door on any potential extensions of logback, like those we are doing for junit, and goes against the maven convention which is to let users define the scope instead of enforcing it at parent level, which might come into surprises and error-prone because we have a mix of implicit scopes like:

```
    <dependency>
      <groupId>org.slf4j</groupId>
      <artifactId>slf4j-api</artifactId>
    </dependency>
    <dependency>
      <groupId>ch.qos.logback</groupId>
      <artifactId>logback-classic</artifactId>
    </dependency>
    <dependency>
      <groupId>org.hamcrest</groupId>
      <artifactId>hamcrest</artifactId>
      <scope>test</scope>
    </dependency>
```

When looking at that, usually, any person would say that first one and second one is compile, last one is test.

But this is not the case because the parent pom is changing the convention of reading a maven bom.

Reply:
Addition: I am not sure this is right to rely on the dependency section to enforce a rule. 
I the maven think the maven enforcer plugin is there for that.

Reply:
Here is an example: https://tenmilesquare.com/utilizing-maven-enforcer-plugin-to-ensure-dependency-and-version-usage/
How to ban: _All versions of Mockito unless it is in the test scope_

Reply:
@cljohnso : added a commit to show how to correctly enforce dependency scopes.